README.md

```diff
@@ -1,26 +1,15 @@
 ## The Golden Rule:
+my steps
 
-🦸 🦸‍♂️ `Stop starting and start finishing.` 🏁
+Step 1 Make title with name/pronouns in bold. 
+step 2 Make where I live under header 
+step 3 create a list with id and populate with favorite animals 
+step 4 create a box for the list 
+step 5 make a paragraph with a fun fact 
+step 6 create box around fun fact 
+step 7 move to css and change site colors 
+step 8 add fonts and font sizes for paragraph and list
+step 9 upload.
 
-If you work on more than one feature at a time, you are guaranteed to multiply your bugs and your anxiety.
 
-## Making a plan
-
-1. **Make a drawing of your app. Simple "wireframes"**
-1. **Once you have a drawing, name the HTML elements you'll need to realize your vision**
-1. **For each HTML element ask: Why do I need this?**
-1. **Once we know _why_ we need each element, think about how to implement the "Why" as a "How"**
-1. **Find all the 'events' (user clicks, form submit, on load etc) in your app. Ask one by one, "What happens when" for each of these events. Does any state change?**
-1. **Think about how to validate each of your features according to a Definition of Done**
-1. **Consider what features _depend_ on what other features. Use this dependency logic to figure out what order to complete tasks.**
-
-Additional considerations:
-
--   Ask: which of your HTML elements need to be hard coded, and which need to be dynamically generated?
--   Consider your data model.
-    -   What kinds of objects (i.e., Dogs, Friends, Todos, etc) will you need?
-    -   What are the key/value pairs?
-    -   What arrays might you need?
-    -   What needs to live in a persistence layer?
--   Is there some state we need to initialize?
--   Ask: should any of this work be abstracted into functions? (i.e., is the work complicated? can it be reused?)
+Things changed and fell out of order but I got it done none the less!
```

Comment:
That is the nature of development, my friend! Things will never go exactly as you have planned them, but the important thing is that you had a basic outline plan established that gave you a path to complete your goal.